routers/api/packages/container/manifest.go

```diff
@@ -437,13 +431,6 @@ func createFileFromBlobReference(ctx context.Context, pv, uploadVersion *package
 		}
 	}
 
-	// Remove the file from the blob upload version
-	if uploadVersion != nil && ref.File.File != nil && uploadVersion.ID == ref.File.File.VersionID {
```

Comment:
@KN4CK3R hello, I can't really understand what this code block does.

1. What is `uploadVersion`? It seems that it is always nil
2. Why `DeletePackageFile` when `uploadVersion.ID == ref.File.File.VersionID`? Won't it delete the inserted properites above?

Reply:
I think I see how it works now, it is something like a "rename from ref file to the new file"?